i18n.md

```diff
@@ -34,7 +34,7 @@ These will output on their own, so an `echo` statement isn't needed.
 
 #### Translation API Abuse
 
-The `esc_html_e` helper functions are sometimes misused as a susbtitute for `echo esc_html`. Always use the second parameter that sets the translation domain. If you don't it could have unanticipated side effects as your strings are mistranslated:
+The `esc_html_e` helper functions are sometimes misused as a substitute for `echo esc_html`. Always use the second parameter that sets the translation domain. If you don't it could have unanticipated side effects as your strings are mistranslated:
```

Comment:
```suggestion
The `esc_html_e` helper functions are sometimes misused as a substitute for `echo esc_html`. This is incorrect and can lead to problems. This is because `esc_html_e` is actually shorthand for `echo esc_html( __(`. You may find your value is localised into something unexpected.

Always use the second parameter that sets the translation domain, and never pass dynamic values into localisation functions. If you don't it could have unanticipated side effects as your strings are mistranslated:
```